src/components/calcite-modal/calcite-modal.tsx

```diff
@@ -311,4 +315,8 @@ export class CalciteModal {
       focusElement(this.closeButtonEl);
     }
   };
+
+  private removeOverflowHiddenClass(): void {
+    document.documentElement.classList.add("overflow-hidden");
```

Comment:
also, aren't you adding here, instead of removing?

Reply:
probably a good idea to use some optional chaining here for environments like prerender that may not have a document:

```
document?.documentElement?.classList.add("overflow-hidden")
```

Reply:
Ha, yeah...